src/data_management/rasters/RasterLayerForm.tsx

```diff
@@ -405,18 +423,21 @@ const RasterLayerForm: React.FC<Props & PropsFromDispatch & RouteComponentProps>
           onFocus={handleFocus}
           onBlur={handleBlur}
           readOnly={!(supplierIds.length > 0 && selectedOrganisation.roles.includes('admin'))}
+          form={"raster_layer_form_id"}
         />
         <div
           className={formStyles.ButtonContainer}
         >
           <CancelButton
             url={'/data_management/rasters/layers'}
+            form={"raster_layer_form_id"}
           />
           <SubmitButton
             onClick={tryToSubmitForm}
+            form={"raster_layer_form_id"}
```

Comment:
Why do you need this form element everywhere? is it to distinguish between the raster layer form and the custom color map form?